setup.py

```diff
@@ -55,7 +55,7 @@
             "codecov",
             "flake8",
             "mock",
-            "pycodestyle<2.4",  # 2.4 is incompatible with flake8
+            "pycodestyle",  # 2.4 is incompatible with flake8
```

Comment:
Pinned elsewhere? Is this flake8 compatibility still an issue?

Reply:
We're just using latest. It doesn't seem that there's an issue anymore. I should have removed the comment.